pkgs/tools/misc/heimdall/default.nix

```diff
@@ -14,10 +14,10 @@ mkDerivation {
     sha256 = "1ygn4snvcmi98rgldgxf5hwm7zzi1zcsihfvm6awf9s6mpcjzbqz";
   };
 
-  buildInputs = [
-    zlib libusb1
-  ] ++ lib.optional enableGUI qtbase;
-  nativeBuildInputs = [ cmake ];
+  buildInputs = [ zlib libusb1 ]
+    ++ lib.optional enableGUI qtbase;
+  nativeBuildInputs = [ cmake ]
+    ++ lib.optional enableGUI wrapQtAppsHook;
```

Comment:
```suggestion
```
I think `wrapQtApps` should already be available due to the use of `libsForQt5.mkDerivation`.

Reply:
Ah, it is! I've removed the other changes to keep the diff small.